trxer.go

```diff
@@ -18,7 +29,161 @@ type measurement struct {
 	time  float64
 }
 
-func udp_client_worker(addr string, wg sync.WaitGroup) {
+func quic_client_worker(addr string, wg *sync.WaitGroup) {
+	fmt.Println("Quic stream connecting to: ", addr)
+
+	buf := make([]byte, 1400, 1400)
+
+	/* create tls conf, true => TLS accepts any certificate presented
+	by the server and any host name in that certificate
+	 */
+	tlsConf := tls.Config {InsecureSkipVerify: true}
+
+	session, err := quic.DialAddr(addr, &tlsConf, nil)
+	if err != nil {
+		panic("dialQuic")
+	}
+
+	defer session.Close()
+
+	// open bidirectional QUIC stream => can be used to open several streams?
+	stream, err := session.OpenStreamSync()
+	if err != nil {
+		panic("openStream")
+	}
+
+	for {
+		_, err := stream.Write(buf)
+		if err !=  nil {
+			panic("writeStream")
+		}
+	}
+
+	defer wg.Done()
+}
+
+func quic_client(threads int, addr string) {
+	port := PORT
+	var wg sync.WaitGroup
+
+	for i := 0;  i < threads; i++ {
+		destAddr := addr + ":" + strconv.Itoa(port)
+		port++
+
+		wg.Add(1)
+		go quic_client_worker(destAddr, &wg)
+	}
+
+	wg.Wait()
+	fmt.Println("Releasing threads")
+}
+
+func quic_server_worker(c chan<- measurement, port int) {
+	var bytesPerInterval uint64 = 0
+	buf := make([]byte, 1400, 1400)
```

Comment:
same here, this should be replaced by BYTE_BUFFER_SIZE.

Or better: make this dynamic again (like in the sending process).

One note: when dynamic argument, then the argument should also work for UDP and TCP.